.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
   - "stable"
-cache: yarn
+install: yarn install
```

Comment:
this fixes an yarn issue https://travis-ci.com/github/FLOIP/flow-builder/builds/219689301#L226
`The command "eval yarn --frozen-lockfile " failed`